brand/README.md

```diff
@@ -0,0 +1,100 @@
+<p align="center"><img src="primary-logo.png" width="360"></p>
+<p align="center"><b>Cloud Native Runtime Security.</b></p>
+
+# Falco Branding Guidelines
+
+This document describes The Falco Project's branding guidelines, language, and message.
+
+Content in this document can be used to publically share about Falco.
+
+
+
+### Logo
+
+There are 3 logos available for use in this directory. Use the primary logo unless required otherwise due to background issues, or printing.
+
+The Falco logo is Apache 2 licensed and free to use in media and publication for the CNCF Falco project.
+
+### Slogan
+
+> Cloud Native Runtime Security
+
+### What is Falco?
+
+Falco is a Runtime Security project originally created by Sysdig, Inc.
+Falco was contributed to the CNCF in October 2018.
+The CNCF now owns The Falco Project.
+
+### What is Runtime Security?
+
+Runtime Security refers to an approach to securing a computer system.
+With Runtime Security an operator deploys **both** prevention tooling (access control, policy enforcement, etc) along side detection tooling (systems observability, anomaly detection, etc).
+Runtime Security is the practice of using detection tooling to detect unwanted behavior, such that it can then be prevented using prevention techniques.
+Runtime Security is a last line of defense, and useful in scenarios where prevention tooling either was unaware of an exploit or attack vector, or when defective applications are ran in even the most secure environment.
+
+### What does Falco do?
+
+Falco consumes signals from the Linux kernel, and container management tools such as Docker and Kubernetes.
+Falco parses the signals and asserts them against security rules.
+If a rule has been violated, Falco triggers an alert. 
+
+### How does Falco work?
+
+Falco uses eBPF (or on older systems a kernel module) to trace syscall events in the kernel.
+Falco enriches these kernel events with information about containers running on the system.
+These other input streams come from various input streams such as the Docker socket, the Kubernetes API server, and the Kubernetes audit log.
+At runtime, Falco will reason about these events and assert them against security rules.
```

Comment:
Asserting these events against the configured security rules, Falco will trigger alerts that can be forwarded to downstream systems, for example to send notifications or start your own workflow.

Reply:
I am terrible at grammar but I think that is a run-on sentence. I added your suggestions about `configured security rules` but kept this broken into 2 sentences.